x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/trainedmodel/FillMaskConfig.java

```diff
@@ -57,6 +62,7 @@ private static ObjectParser<FillMaskConfig.Builder, Void> createParser(boolean i
         );
         parser.declareInt(Builder::setNumTopClasses, NUM_TOP_CLASSES);
         parser.declareString(Builder::setResultsField, RESULTS_FIELD);
+    //  tried to add parser for mask token, but caused »  Caused by: java.lang.IllegalArgumentException: [parseField] is require
```

Comment:
What was the exact line you had here?

Was it by any chance `parser.declareString(Builder::setMaskToken, MASK_TOKEN);`? If so then the problem would be that the `ParseField` is called `MASK_TOKEN_FIELD`.

Reply:
it turned out to be a different issue. I had declared the ParseField after I had declared the Parser, and because they were both static the ParseField was null when the Parser was initialized.